modules/connectivity/NetworkInfo.js

```diff
@@ -26,7 +26,6 @@ export class NetworkInfo extends Listenable {
 
     /**
      * Updates the network info state.
-     * @param {boolean} isOnline - {@code true} if internet is online or {@code false} otherwise.
```

Comment:
Why was this removed?

Reply:
It's incorrectly defined as a Boolean not an object containing a Boolean. TS guesses it right.